vendor/github.com/docker/distribution/README.md

```diff
@@ -2,7 +2,7 @@
 
 The Docker toolset to pack, ship, store, and deliver content.
 
-This repository's main product is the Docker Registry 2.0 implementation
+This repository provides the Docker Registry 2.0 implementation
```

Comment:
We are not supposed to modify the vendor directory manually :)

Reply:
fixed